explore-htap.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Explore HTAP
+summary: Learn how to explore and use the features of TiDB HTAP.
+---
+
+# Explore HTAP
+
+This guide describes how to explore and use the features of TiDB Hybrid Transactional and Analytical Processing (HTAP).
+
+> **Note:**
+>
+> If you are new to TiDB HTAP and want to start using it quickly, see [Quick start with HTAP](/quick-start-with-htap.md).
+
+## Use cases
+
+TiDB HTAP can meet the needs that have increment massive data, reduce the risk cost of operation, and be used for on-premises big data technology stacks without difficulty, thereby presenting the value of data assets in real time.
+
+The following are the use cases of HTAP:
+
+- Hybrid load
+
+    When using TiDB for real-time Online Analytical Processing (OLAP) that is in hybrid load scenarios, you only need to provide an entry point. Then TiDB automatically selects different processing engines based on the specific business.
```

Comment:
`you only need to provide an entry point` -> It's mostly user-transparent and don't require additional work of users